httpbp/handler.go

```diff
@@ -59,8 +59,8 @@ var (
 // instead.
 // NewHander is provided for those who need to avoid the default Baseplate
 // middleware or for testing purposes.
-func NewHandler(handle HandlerFunc, middlewares ...Middleware) http.Handler {
-	return handler{handle: Wrap(handle, middlewares...)}
+func NewHandler(handle HandlerFunc, name string, middlewares ...Middleware) http.Handler {
```

Comment:
style question, I feel ilke maybe `name` should come _first_ in these functions and in `Wrap`?  It lines up with the `Handle` functions on `http.ServeMux` that way too.

Reply:
yea name being the first one does look better.

Reply:
But I think we need to mention that this is just a name, it does not affect routing, as people might get confused.